frontend/src/app/testResults/uploads/Uploads.tsx

```diff
@@ -143,91 +147,123 @@ const Uploads = () => {
+                    <thead>
+                      <tr>
+                        <th>Requested Edit</th>
+                        <th>Areas Containing the Requested Edit</th>
+                      </tr>
+                    </thead>
+                    <tbody>
+                      {errors.map((e, i) => {
+                        return (
+                          <tr key={"error_" + i}>
+                            <td>{e?.["message"]} </td>
+                            <td>Row(s): {e?.["indices"]}</td>
+                          </tr>
+                        );
+                      })}
+                    </tbody>
+                  </table>
                 </div>
-                <table className="usa-table usa-table--borderless">
-                  <thead>
-                    <tr>
-                      <th>Requested Edit</th>
-                      <th>Areas Containing the Requested Edit</th>
-                    </tr>
-                  </thead>
-                  <tbody>
-                    {errors.map((e, i) => {
-                      return (
-                        <tr key={"error_" + i}>
-                          <td>{e?.["message"]} </td>
-                          <td>Row(s): {e?.["indices"]}</td>
-                        </tr>
-                      );
-                    })}
-                  </tbody>
-                </table>
-              </div>
-            )}
-            <FormGroup className="margin-bottom-3">
-              <Label
-                className="font-sans-xs"
-                id="upload-csv-input-label"
-                htmlFor="upload-csv-input"
-              >
-                Upload your COVID-19 lab results as a .csv.
-              </Label>
-              <FileInput
-                key={fileInputResetValue}
-                id="upload-csv-input"
-                name="upload-csv-input"
-                aria-describedby="upload-csv-input-label"
-                accept="text/csv, .csv"
-                onChange={(e) => handleFileChange(e)}
-                required
-              />
-            </FormGroup>
-            <Button
-              type="submit"
-              onClick={(e) => handleSubmit(e)}
-              disabled={buttonIsDisabled || file?.name?.length === 0}
-            >
-              {isSubmitting && (
-                <span>
-                  <span>Processing file...</span>
-                </span>
               )}
-              {!isSubmitting && <span>Upload</span>}
-            </Button>
+              <FormGroup className="margin-bottom-3">
+                <Label
+                  className="font-sans-xs usa-hint"
+                  id="upload-csv-input-label"
+                  htmlFor="upload-csv-input"
+                >
+                  Upload your test results as a spreadsheet. You can create a
+                  .csv file in any spreadsheet application like Excel or Google
+                  Sheets
+                </Label>
```

Comment:
I'd recommend removing this for now, since most of the information is covered in the steps above (added the spreadsheet application piece to the first bullet).

Reply:
Makes sense! Removed in [5f3dcc5](https://github.com/CDCgov/prime-simplereport/pull/3928/commits/5f3dcc516c9a34235bf87ebfd2d9aacecbde6921).